bikeshed/boilerplate.py

```diff
@@ -715,4 +726,3 @@ def addIssuesSection(doc):
         del idel.attrib['id']
     for dfnel in findAll(config.dfnElementsSelector, container):
         dfnel.tag = "span"
-
```

Comment:
Lost the final linebreak here. I'll just put it back next time I touch this file, so let's avoid diff churn. ^_^